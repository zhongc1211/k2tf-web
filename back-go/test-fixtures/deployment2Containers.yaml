apiVersion: extensions/v1beta1
kind: Deployment
metadata:
  name: backend-api
spec:
  selector:
    matchLabels:
      app: backend-api
  template:
    metadata:
      labels:
        app: backend-api
    spec:
      containers:
        - args:
            - --ssl_port
            - "443"
          image: gcr.io/container1/image:latest
          name: esp
          ports:
            - containerPort: 443
              protocol: TCP
        - command:
            - /root/backend-api
          image: gcr.io/container2/image:latest
          name: api
