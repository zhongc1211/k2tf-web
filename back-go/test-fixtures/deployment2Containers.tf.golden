resource "kubernetes_deployment" "backend_api" {
  metadata {
    name = "backend-api"
  }
  spec {
    selector {
      match_labels = { app = "backend-api" }
    }
    template {
      metadata {
        labels = { app = "backend-api" }
      }
      spec {
        container {
          name  = "esp"
          image = "gcr.io/container1/image:latest"
          args  = ["--ssl_port", "443"]
          port {
            container_port = 443
            protocol       = "TCP"
          }
        }
        container {
          name    = "api"
          image   = "gcr.io/container2/image:latest"
          command = ["/root/backend-api"]
        }
      }
    }
  }
}
